.github/workflows/go-build.yaml

```diff
@@ -15,23 +15,22 @@
 name: Build
 
 on:
+  push:
+    branches: [ 'master', 'release-*' ]
   pull_request:
-    branches: [ master ]
+    branches: [ 'master', 'release-*' ]
 
 jobs:
 
   build:
     name: Build
     strategy:
       matrix:
-        go-version: [1.14.x]
+        go-version: [1.15.x]
```

Comment:
Should we do both?

Reply:
I believe that generally the toolchain keys off of the version in go.mod, so it felt like a waste.  I also figure we'll want 1.15 by the time we're using these for realz.